app/Http/Controllers/BudgetsController.php

```diff
@@ -38,28 +41,18 @@ public function create()
      */
     public function store(Request $request)
     {
+
         $this->validate($request, [
             'name' => 'required',
-            'money' => 'required',
         ]);
 
-        Budget::create($request->all());
-
+        $budget = Budget::create($request->all());
+        $budget->user_id = Auth::user()->id;
+        //$budget->money = 0;
+        $budget->save();
```

Comment:
```suggestion
        Budget::create(array_merge($request->validate([
           'name' => ['required', 'string'],
           'money' => ['required', 'numeric', 'min:0']
        ]), [
           'user_id' => auth()->id()
        ]));
```
Но лучше валидацию вынести в реквест класс и применить его тут